connector_prestashop_catalog_manager/models/product_template/common.py

```diff
@@ -2,10 +2,24 @@
 # © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from openerp import models, fields
+from openerp import api, models, fields
 import openerp.addons.decimal_precision as dp
 
 
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    def _set_multi_image_main_medium(self):
```

Comment:
The main image of a product is exported three times when it's modified:

![jobs](https://user-images.githubusercontent.com/39995504/46289374-35497c80-c589-11e8-92ae-9670ac29480d.png)

This is due to the design of module `product_multi_image`.

Each export involves the removal and creation again of the image in Prestashop and concurrent exports can fail and break the binding.

The export of medium and small images can be avoided.